origin-js/src/contractInterface/users/v00_adapter.js

```diff
@@ -116,7 +117,12 @@ class V00_UsersAdapter {
             return data.substr(2)
           })
           .join('')
-      const dataOffsets = attestations.map(() => 32) // all data hashes will be 32 bytes
+
+      /* All the data is currenlty base32 encoded. To calculate the lenght we just deduct 2 (becase '0x') gets
```

Comment:
nit: same as previous comment. The data binary and since it is hex encoded, it is twice as long as in its binary encoded form.

Reply:
👍 changing the comment to: 
```
All the data is in bytes32 binary format in the contract. To calculate the lenght we just
deduct 2 (becase '0x') gets removed. And then divide the remaining lenght by 2 becase
the data is twice the size when it is represented as string (with hex encoding).
```